saathi.sql

```diff
@@ -63,7 +63,8 @@ CREATE TABLE IF NOT EXISTS `contact_details` (
   `email` varchar(255) NOT NULL,
   `mailing_list` varchar(255) NOT NULL,
   `mailing_address` varchar(255) NOT NULL,
-  `gis_url` varchar(255) NULL,
+  `longitude` varchar(255) NULL,
+  `latitude` varchar(255) NULL,
```

Comment:
Can't we have just one field for this? Also, the `varchar(255)` seems odd, please check.